app/src/main/AndroidManifest.xml

```diff
@@ -73,6 +73,16 @@
 
             </intent-filter>
 
+            <intent-filter android:label="@string/app_name">
```

Comment:
What about autoVerify, this deep link did you tested it on Android 13 ?

Reply:
it is app link not an http so no need it autoVerify
but i will recheck thx

Reply:
yes checked it
works fine on Android 13